bash/bash-quiz.md

```diff
@@ -717,7 +718,7 @@ textfile1.txt
 textfile2.txt
 ```
 
-- [ ] `c : All of this files will be deleted`
+- [ ] ```c : All of this files will be deleted```
```

Comment:
```suggestion
- [ ] `c : All of this files will be deleted`
```

fyi it is fine to keep with one backtick

Reply:
I'll be honest, I don't remember adding those double back ticks. You are well within your reason and right to change it back.